tools/ci_build/github/azure-pipelines/templates/py-packaging-selectable-stage.yml

```diff
@@ -22,12 +22,12 @@ parameters:
 - name: python_version
   displayName: 'Python version used for build'
   type: object
-  default: ['3.7', '3.8', '3.9']
+  default: ["'3.7'", "'3.8'", "'3.9'"]
 
 - name: cpu_build_py_parameters
   displayName: 'Extra parameters to pass to build.py for CPU package.'
   type: string
-  default: '--use_dnnl --use_openvino CPU_FP32'
```

Comment:
Why do you need to remove dnnl?

Reply:
Anubis uses this pipeline more frequently and they only need openvino. 
dnnl ep can be set when running this pipeline.